[2023-03-22 07:54:03,944] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: load_to_datamart.Start_Proses_Load_to_datamart manual__2023-03-22T07:54:02.325409+00:00 [queued]>
[2023-03-22 07:54:03,983] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: load_to_datamart.Start_Proses_Load_to_datamart manual__2023-03-22T07:54:02.325409+00:00 [queued]>
[2023-03-22 07:54:03,984] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2023-03-22 07:54:03,984] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2023-03-22 07:54:03,985] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2023-03-22 07:54:04,008] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): Start_Proses_Load_to_datamart> on 2023-03-22 07:54:02.325409+00:00
[2023-03-22 07:54:04,014] {standard_task_runner.py:52} INFO - Started process 302 to run task
[2023-03-22 07:54:04,020] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'load_to_datamart', 'Start_Proses_Load_to_datamart', 'manual__2023-03-22T07:54:02.325409+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/mart_layer.py', '--cfg-path', '/tmp/tmp1igrqffy', '--error-file', '/tmp/tmpb2qjvzun']
[2023-03-22 07:54:04,021] {standard_task_runner.py:77} INFO - Job 9: Subtask Start_Proses_Load_to_datamart
[2023-03-22 07:54:04,095] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_datamart.Start_Proses_Load_to_datamart manual__2023-03-22T07:54:02.325409+00:00 [running]> on host c456fd334a3e
[2023-03-22 07:54:04,175] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_datamart
AIRFLOW_CTX_TASK_ID=Start_Proses_Load_to_datamart
AIRFLOW_CTX_EXECUTION_DATE=2023-03-22T07:54:02.325409+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-22T07:54:02.325409+00:00
[2023-03-22 07:54:04,176] {logging_mixin.py:109} INFO - ETL Dimulai
[2023-03-22 07:54:04,177] {python.py:152} INFO - Done. Returned value was: None
[2023-03-22 07:54:04,198] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=load_to_datamart, task_id=Start_Proses_Load_to_datamart, execution_date=20230322T075402, start_date=20230322T075403, end_date=20230322T075404
[2023-03-22 07:54:04,232] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-22 07:54:04,309] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
